perceval/backends/_naive.py

```diff
@@ -67,5 +67,5 @@ def _compute_submatrix(self, output_state: BasicState) -> np.ndarray:
                 colidx += 1
         return u_st
 
-    def _compute_permanent(self, M: np.ndarray):
-        return xq.permanent_cx(M, n_threads=1)
+    def _compute_permanent(self, m):
```

Comment:
why was the typehinting for the argument `m` removed?

Reply:
Because it was not accurate enough, and the actual type is hard to express.